cardano-client/cardano-client.cabal

```diff
@@ -22,9 +22,10 @@ library
                        bytestring        >=0.10 && <0.11,
                        containers,
                        io-classes,
-                       ouroboros-consensus,
+                       ouroboros-network-api,
                        ouroboros-network,
                        ouroboros-network-framework,
+                       ouroboros-node,
```

Comment:
Is `ouroboros-node` perhaps too general of a name? My line of thought:

> there are a _lot_ of things that the `cardano-node` package handles for Cardano that this `ouroboros-node` does _not_ handle for a general node based in Ouroboros

Reply:
I renamed it to `ouroboros-consensus-diffusion`.